scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -694,7 +694,10 @@ def handle(self, harness):
             timeout = 30
             if script_param:
                 timeout = script_param.get("pre_script_timeout", timeout)
-            self.run_custom_script(pre_script, timeout)
+            pre_script_cmd = ([pre_script] +
```

Comment:
please document this new behavior for `--pre-script` command

https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/pylib/twister/twisterlib/environment.py#L627-L630

Reply:
I will add